### DISCUSSION TITLE
Multiple ES versions on `env`

### DISCUSSION THREAD

**Discussion by arslivinski** [eslint/eslint#16725] [Q&A] [answered]
Today I'm doing this:
```
env: {
  es6: true,
  es2016: true,
  es2017: true,
  es2018: true,
  es2019: true,
  es2020: true,
  es2021: true,
  es2022: true,
},
```
Is this required? Or only setting `es2022` will set all previous versions too?

**Comment by mdjermanovic** [comment, marked as answer]
`es2022: true` will set `ecmaVersion: 2022` parsing and enable all ES2022 globals including < 2022. There's no need to also enable es6...es2021.
